src/components/ForecastSummaries.js

```diff
@@ -1,39 +0,0 @@
-import React from "react";
-import PropTypes from "prop-types";
-import ForecastSummary from "./ForecastSummary";
-import "../styles/ForecastSummaries.css";
-
-function ForecastSummaries({ forecasts, onForecastSelect }) {
-  return (
-    <div className="forecast-summaries">
-      {forecasts.map((forecast) => (
-        <ForecastSummary /* this is the list of props that we are passing the ones that we receive are in the brackets above */
-          key={forecast.date}
-          date={forecast.date}
-          description={forecast.description}
-          icon={forecast.icon.toString()}
-          temperature={forecast.temperature}
-          onSelect={
-            onForecastSelect
-          } /* Here we are passing the eventhandler from app to Forecast summaries and now from here to forecast summary ,note also that the function is included in the validation */
-        />
-      ))}
```

Comment:
You can destructure these props down further so you don't have to do `forecast.` every time.

One way is to destructure them on line 7 just below the definition of the function:
```
function ForecastSummaries({forecast, onForecastSelect}) {
  const {date, description, icon, temperature} = forecast
  return{
          <ForecastSummary 
            key={date}
            date={date}
            description={description}
            icon={icon.toString()}
            temperature={temperature}
            onSelect={
            onForecastSelect
          } 
        />
```
Or you can do it in the brackets like this:
```
function ForecastSummaries({forecast: {date, description, icon, temperature}, onForecastSelect}) {
  return{
          <ForecastSummary 
            key={date}
            date={date}
            description={description}
            icon={icon.toString()}
            temperature={temperature}
            onSelect={
            onForecastSelect
          } 
        />
```